roles/zuul/files/jobs/dib.yaml

```diff
@@ -0,0 +1,26 @@
+- job:
+    name: 'dib-build'
+    node: 'ubuntu-xenial'
+    builders:
+      shell: |
+        virtualenv $HOME/dib-build
+        source $HOME/dib-build/bin/activate
+        pip install diskimage-builder
+
+        sudo apt-get install -y debootstrap qemu-utils curl \
+                                uuid-runtime parted kpartx
+
+        sudo git clone https://github.com/BonnyCI/hoist.git /opt/hoist
```

Comment:
So cloning and using the upstream repository means we wouldn't actually be testing the proposed changes here, and instead using whats already been merged.  When the test runs, it'll be running from the root of the hoist repo with the proposed PR already checked out, so you should be able to just point ELEMENTS_PATH to ./roles/nodepoole/files/etc/nodepool/elements/